docs/mkdocs/en/evolution.md

```
@@ -252,6 +252,53 @@ The decision is stored on `HumanReport` (`Approved`, `Reviewer`, `Comment`,
+    Comment:  "regressed quality",
+    // TargetRevisionID: "20260601T120000.000-abc123", // optional: pick a specific archived revision
+})
+fmt.Printf("rolled back %s → %s\n", res.PreviousActiveID, res.RestoredID)
+```
+
+When `TargetRevisionID` is empty the most recently archived revision wins.
+`ErrNoArchivedRevision` is returned when no archived revision is available.
+
+The `openclaw` CLI exposes the same workflow:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)